dallinger/command_line/docker_ssh.py

```diff
@@ -235,6 +233,18 @@ def copy_docker_config(host, user):
     default=False,
     help="Update an existing experiment",
 )
+option_remote_build = click.option(
```

Comment:
The remote build option is so great, I would say we should make it the default, and then have people specify `--local-build` if they really want to build it locally.